sites/partners/src/lib/hooks.ts

```diff
@@ -521,6 +521,15 @@ export const useApplicationsExport = (listingId: string, includeDemographics: bo
   )
 }
 
+export const useLotteryExport = (listingId: string) => {
+  const { applicationsService } = useContext(AuthContext)
+
+  return useCsvExport(
+    () => applicationsService.lotteryResults({ listingId, timeZone: dayjs.tz.guess() }),
```

Comment:
Looking at the sample export, looks like we want to have demographic info on the export https://docs.google.com/spreadsheets/d/1dX9ehAq3pdMpmhNq1arWV2bwjHBFMy6xAbcPGhLVHOY/edit?gid=1191380628#gid=1191380628. You can just add it as a query param here

Reply:
if its always going to be on I think it may be better to force it on the api side so I updated the call to have `includeDemographics: true` always